Packs/AzureNetworkSecurityGroups/Integrations/AzureNetworkSecurityGroups/AzureNetworkSecurityGroups.py

```diff
@@ -134,6 +135,37 @@ def list_resource_groups_request(self, subscription_id: str | None,
                                            params={'$filter': filter_by_tag, '$top': limit,
                                                    'api-version': '2021-04-01'})
 
+    @logger
+    def list_public_ip_addresses(self, subscription_id: str, resource_group_name: str) -> Dict:
```

Comment:
```suggestion
    def list_public_ip_addresses_request(self, subscription_id: str, resource_group_name: str) -> Dict:
```
Rename all request functions to end with `_request`